eng/common/templates/steps/test-images-windows-client.yml

```diff
@@ -1,9 +1,12 @@
 steps:
 - powershell: |
     if ("${{ eq(variables['System.TeamProject'], 'public') }}" -eq "False") {
-      $optionalTestArgs="-Registry $env:ACR_SERVER -RepoPrefix $env:STAGINGREPOPREFIX -TestCategory $env:TESTCATEGORY"
+      $optionalTestArgs="-Registry $env:ACR_SERVER -RepoPrefix $env:STAGINGREPOPREFIX"
     } else {
-      $optionalTestArgs="-IsLocalRun -TestCategory $env:TESTCATEGORY"
+      $optionalTestArgs="-IsLocalRun"
+    }
+    if ($repoTestArgs) {
```

Comment:
Does this need to be using `$env:REPOTESTARGS`?

Reply:
Possibly, not very familiar with PS, I'll test it in azdo to confirm.